client/version/repository.go

```diff
@@ -34,10 +34,11 @@ func (ar s3assetRepository) All() ([]Asset, error) {
 	var assets []Asset
 	scanner := bufio.NewScanner(response.Body)
 	for scanner.Scan() {
-		version, err := filenameParser(scanner.Text())
+		filename := scanner.Text()
+		version, err := filenameParser(filename)
 		if err != nil {
 			// ignore error and continue to next filename
-			log.Debug("could not parse filename", "err", err)
+			log.Debug("file is not a binary asset: can't parse filename", "filename", filename)
```

Comment:
I think you should keep the `err` in the debug message

Reply:
maybe with `"message"`?